bower.json

```diff
@@ -15,5 +15,8 @@
   "ignore": [
     "**/.*",
     "!dist/braintree-angular.js"
-  ]
+  ],
+  "dependencies": {
+    "braintree-web": "2.5.3"
+  }
```

Comment:
@veswill3 I'm sorry for taking so long to get back to you on this :(. Unless you're building the module (all you're doing is using `dist/braintree-angular.js`), you shouldn't need to install `braintree-web` as a dependency. Since bower isn't being used in development, I don't think it needs to live in devDependencies either.